src/views/Components/PlayerSettings.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import Button from '../../elements/Button'
+import arrowButton from '../../assets/svg-icons/arrow.svg';
+
+function Settings() {
+    function createPlayers() {
+        return [
+            {
+                id: 0,
+                name: "Player 1",
+            },
+            {
+                id: 1,
+                name: "Player 2",
+            },
+            {
+                id: 2,
+                name: "Player 3",
+            },
+            {
+                id: 3,
+                name: "Player 4",
+            },
+        ];
+    }
+    const [playerNames, setPlayerNames] = useState(createPlayers());
```

Comment:
Would it make more sense just to be called `players` instead of `playerNames`? It's not quite a list of names, it's a list of player objects. What do you think about this?

Reply:
That makes sense. I was thinking about it from a UI perspective where it shows them the names of the players. But in the code it makes more sense to not specify it is just names.